cpp/src/cylon/table.cpp

```diff
@@ -1756,5 +1770,191 @@ Status WriteParquet(const std::shared_ptr<cylon::CylonContext> &ctx_,
 
   return Status(Code::OK);
 }
+
+/**
+ * Local_Slice the part of table to create a single table
+ * @param table, offset and length
+ * @return new sliced table
+ */
+
+
+Status Local_Slice(const std::shared_ptr<Table> &in, int64_t offset, int64_t length,
+              std::shared_ptr<cylon::Table> &out) {
+  const auto &ctx = in->GetContext();
+  std::shared_ptr<arrow::Table> out_table, in_table = in->get_table();
+
+  if (!in->Empty()) {
+    out_table = in_table->Slice(offset, length);
+  } else {
+    out_table = in_table;
+  }
+  return Table::FromArrowTable(ctx, std::move(out_table), out);
+}
+
+
+/**
+ * Distributed_Slice the part of table to create a single table
+ * @param table, offset and length
+ * @return new sliced table
+ */
+
+
+Status Distributed_Slice(const std::shared_ptr<cylon::Table> &in, int64_t offset, int64_t length,
+              std::shared_ptr<cylon::Table> &out, int order) {
```

Comment:
based on our previous discussion, we need to put check to verify that `offset` and `length` are the same in all partitions.

Reply:
offsets and length are coming from the main functions, mpirun creates individual operations for each partition. So, two values will remain the same.